openlibrary/plugins/upstream/tests/test_merge_authors.py

```diff
@@ -1,5 +1,10 @@
+import web
+
 from infogami.infobase import client, common
-from openlibrary.plugins.upstream.merge_authors import *
+from openlibrary.plugins.upstream.merge_authors import make_redirect_doc, \
+    BasicRedirectEngine, BasicMergeEngine, \
+    get_many, AuthorRedirectEngine, space_squash_and_strip, AuthorMergeEngine
```

Comment:
Please wrap imports with parens () instead of backslashes as discussed in PEP8.  The problem with backslashes is that any whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.

Reply:
Fixed